sdk/monitor/monitor-opentelemetry/package.json

```diff
@@ -89,7 +89,7 @@
     "@azure/logger": "^1.0.0",
     "@azure/monitor-opentelemetry-exporter": "1.0.0-beta.24",
     "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.5",
-    "@microsoft/applicationinsights-web-snippet": "^1.1.2",
+    "@microsoft/applicationinsights-web-snippet": "1.2.1",
```

Comment:
Do you want to pin to 1.2.1 version?

Reply:
Typo in the `rush add` command. Fixing that now.